setup_ingest_config.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
```

Comment:
would be nice to download from the ebi-ait upload service repo

Reply:
Maybe but if the Hinxton CLI is available soon I think it's simpler to just copy in the script